src/Administration/Resources/app/administration/src/module/sw-order/page/sw-order-list/index.js

```diff
@@ -560,30 +540,17 @@ export default {
             });
         },
 
+        /**
+         * @deprecated tag:v6.8.0 - will be removed, use order.primaryOrderTransaction instead
+         */
         transaction(item) {
-            for (let i = 0; i < item.transactions.length; i += 1) {
-                if (
-                    ![
-                        'cancelled',
-                        'failed',
-                    ].includes(item.transactions[i].stateMachineState.technicalName)
-                ) {
-                    return item.transactions[i];
-                }
-            }
-
-            if (!Shopware.Feature.isActive('v6.8.0.0')) {
-                return item.transactions.last();
-            }
-
             return item.primaryOrderTransaction;
```

Comment:
That was not my intension. We need to keep the Feature check, but

```suggestion
          if (!Shopware.Feature.isActive('v6.8.0.0')) {
              for (let i = 0; i < item.transactions.length; i += 1) {
                    if (
                        ![
                            'cancelled',
                            'failed',
                        ].includes(item.transactions[i].stateMachineState.technicalName)
                    ) {
                        return item.transactions[i];
                    }
                }
          
                return item.transactions.last();
            }

            return item.primaryOrderTransaction;
```

Otherwise, this will be a BC